app/views/feedbacks/new.html.erb

```diff
@@ -17,7 +17,7 @@
 <!--main content-->
 <h3 class="heading"><%= ts("We love to hear from our users! Tell us about your experience - good, bad, or broken - using our site!") %></h3>
 
-<p><%= ts("We're still in our beta phase of development, so there are ") %><a href="https://otwarchive.atlassian.net/projects/AO3/issues"><%= ts("lots of bugs to be fixed") %></a><%= ts(" and ") %><a href="https://trello.com/board/feature-requests/4f0cd935f97f91cf29246288"><%= ts("new features to be considered") %></a><%= ts(". For live updates on Archive performance, please check out our Twitter feed: ") %><a href="http://twitter.com/ao3_status">@AO3_status</a><%= ts(". Please use this form to let us know about your experience while using the site - good and bad! This form is for questions about the Archive and the OTW. Questions for authors should be left on their works. If you have technical issues with this page, please use ") %><a href="http://transformativeworks.org/contact/archive%20support"><%= ts("the back-up form.") %></a></p>
+<p><%= ts("We're still in our beta phase of development, so there are ") %><a href="https://otwarchive.atlassian.net/projects/AO3/issues"><%= ts("lots of bugs to be fixed") %></a><%= ts(" and ") %><a href="https://trello.com/board/feature-requests/4f0cd935f97f91cf29246288"><%= ts("new features to be considered") %></a><%= ts(". For live updates on Archive performance, please check out our Twitter feed: ") %><a href="http://twitter.com/ao3_status">@AO3_status</a><%= ts(". Please use this form to let us know about your experience while using the site - good and bad! This form is for questions about the Archive and the OTW. Questions for authors should be left on their works. If you have technical issues with this page, please use ") %><a href="http://www.transformativeworks.org/contact_us/contact-archive-support/"><%= ts("the back-up form.") %></a></p>
```

Comment:
Could you make this one long .html_safe string like the paragraph below? (I also changed "authors" to "creators," and you'll need to check my escaping of the apostrophe -- I can never remember where it goes without Googling.)

```
<p><%= ts('We\'re still in our beta phase of development, so there are <a href="https://otwarchive.atlassian.net/projects/AO3/issues">lots of bugs to be fixed</a> and <a href="https://trello.com/board/feature-requests/4f0cd935f97f91cf29246288">new features to be considered</a>. For live updates on Archive performance, please check out our Twitter feed: <a href="http://twitter.com/ao3_status">@AO3_status</a>. Please use this form to let us know about your experience while using the site - good and bad! This form is for questions about the Archive and the OTW. Questions for creators should be left on their works. If you have technical issues with this page, please use <a href="http://www.transformativeworks.org/contact_us/contact-archive-support/">the back-up form</a>.').html_safe %></p>
```